libtenzir/include/tenzir/series.hpp

```diff
@@ -19,11 +20,15 @@
 namespace tenzir {
 
 /// A temporary series representation (until we have a proper one).
```

Comment:
```suggestion
```

Reply:
Instead of removing, I chose to replace this with a meaningful description instead.